source/particle/interpolator/bilinear_least_squares.cc

```diff
@@ -51,6 +54,18 @@ namespace aspect
         AssertThrow(selected_properties.n_selected_components(n_particle_properties) == 1,
                     ExcNotImplemented("Interpolation of multiple components is not supported."));
 
+        if (use_global_valued_limter)
+          {
+            AssertThrow(global_minimum_particle_properties.size() == n_particle_properties,
+                        ExcMessage("Make sure that the size of 'Global minimum particle property' is equivalent to the number "
+                                   "of particle properties."));
+
+            AssertThrow(global_maximum_particle_properties.size() == n_particle_properties,
+                        ExcMessage("Make sure that the size of 'Global maximum particle property' is equivalent to the number "
+                                   "of particle properties."));
+
+          }
```

Comment:
Yes, you will need to add the following includes (if they are not already included in the .cc or .h file) to make that work:
```
#include <aspect/particle/world.h>
#include <aspect/simulator.h>
```